### PR TITLE
Initial README, with sections commented out that aren't yet true

### PR DESCRIPTION
**Pull request by mthaddon**
Applicable spec: N/A

### Overview

A basic README.

### Rationale

Update the README.

### Juju Events Changes

N/A
### Module Changes

N/A

### Library Changes

N/A

### Checklist

- [x] The [charm style guide](https://juju.is/docs/sdk/styleguide) was applied
- [x] The [contributing guide](https://github.com/canonical/is-charms-contributing-guide) was applied
- [x] The changes are compliant with [ISD054 - Managing Charm Complexity](https://discourse.charmhub.io/t/specification-isd014-managing-charm-complexity/11619)
- [x] The documentation is generated using `src-docs`
- [ ] The documentation for charmhub is updated
- [x] The PR is tagged with appropriate label (`urgent`, `trivial`, `complex`)

No charmhub docs yet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,26 +1,48 @@
...
+[![Publish to edge](https://github.com/canonical/penpot-operator/actions/workflows/publish_charm.yaml/badge.svg)](https://github.com/canonical/penpot-operator/actions/workflows/publish_charm.yaml)
+[![Promote charm](https://github.com/canonical/penpot-operator/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/penpot-operator/actions/workflows/promote_charm.yaml)
+[![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
 -->
 
-# is-charms-template
-
-Charmhub package name: operator-template
-More information: https://charmhub.io/is-charms-template
+A Juju charm deploying and managing Penpot on Kubernetes. Penpot is the
```
Comment (merkata) [inline README.md:8]: I would add a link to penpot like

```suggestion
A Juju charm deploying and managing [Penpot](https://penpot.app) on Kubernetes. Penpot is the
```

**Review by merkata — DISMISSED** [review]

**Review by nrobinaubertin — DISMISSED** [review]

File: README.md [hunk 2]
```diff
@@ -1,26 +1,48 @@
...
+such as scaling the number of instances, integration with external
+authentication providers, access to S3 for redundant file storage and more. It
+allows for deployment on many different Kubernetes platforms, from [MicroK8s](https://microk8s.io) to
+[Charmed Kubernetes](https://ubuntu.com/kubernetes) to public cloud Kubernetes
+offerings.
+-->
 
-## Other resources
+As such, the charm makes it easy for those looking to take control of their
+own Penpot deployment while keeping operations sumple, and gives them the
```
Comment (erinecon) [inline README.md:22]: sumple --> simple

**Review by merkata — DISMISSED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (mthaddon) [reply README.md]: Good catch, thx

File: README.md [hunk 3]
```diff
@@ -1,26 +1,48 @@
...
+[![Publish to edge](https://github.com/canonical/penpot-operator/actions/workflows/publish_charm.yaml/badge.svg)](https://github.com/canonical/penpot-operator/actions/workflows/publish_charm.yaml)
+[![Promote charm](https://github.com/canonical/penpot-operator/actions/workflows/promote_charm.yaml/badge.svg)](https://github.com/canonical/penpot-operator/actions/workflows/promote_charm.yaml)
+[![Discourse Status](https://img.shields.io/discourse/status?server=https%3A%2F%2Fdiscourse.charmhub.io&style=flat&label=CharmHub%20Discourse)](https://discourse.charmhub.io)
 -->
 
-# is-charms-template
-
-Charmhub package name: operator-template
-More information: https://charmhub.io/is-charms-template
+A Juju charm deploying and managing [Penpot](https://penpot.app) on Kubernetes. Penpot is the
```
Comment (erinecon) [inline README.md:8]: Nitpick: The first sentence reads a bit strange to me. I would personally rephrase to say either "A Juju charm that deploys and manages Penpot..." or "A Juju charm to deploy and manage Penpot...".

In any event, it doesn't bother me too much.

**Review by Thanhphan1147 — DISMISSED** [review]

File: README.md (hunk 3 above)
Comment (mthaddon) [reply README.md]: I like it too, have changed this.

**Review by erinecon — APPROVED** [review]

**Review by gregory-schiano — APPROVED** [review]
LGTM

**Review by nrobinaubertin — APPROVED** [review]

**Pull request merged**